code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanup.java

```diff
@@ -309,6 +325,13 @@ private void cleanup(String name, List<CleanableTable> tables, Date cutoffTime)
                     log.info(e.getMessage());
                     break;
                 }
+                if (idsToFix.size() > 0) {
+                    ids = create()
+                        .select(id)
+                        .from(table.table)
+                        .where(remove.lt(cutoffTime))
+                        .limit(QUERY_LIMIT_ROWS.getValue()).offset(idsToFix.size());
```

Comment:
I think there is technically no guarantee of order when doing a query and I've actually recently ran some queries against cattle dbs where the ids were in fact out of order. So, should we add an order by id clause to this query?